Document registration update behavior (#170)
rancher/elemental-docs: docs/customizing.md

#### Custom Elemental client configuration file

> [Elemental client](https://github.com/rancher/elemental-toolkit/blob/main/docs/elemental.md) `install`, `upgrade` and `reset` commands can be configured with a custom [configuration file](https://rancher.github.io/elemental-toolkit/docs/customizing/general_configuration/) located in the `/elemental/config.yaml`.  
> Multiple yaml files inside the `/elemental/config.d` folder will be loaded too.  

Reviewer: ```suggestion
[Elemental client](https://github.com/rancher/elemental-toolkit/blob/main/docs/elemental.md) `install`, `upgrade` and `reset` commands can be configured with a [custom configuration file](https://rancher.github.io/elemental-toolkit/docs/customizing/general_configuration/) located by default in `/elemental/config.yaml` or, if you have multiple yaml files, the `/elemental/config.d` directory will be loaded too.  
```
